plugins/opds.koplugin/main.lua

```diff
@@ -64,7 +66,7 @@ end
 function OPDS:addToMainMenu(menu_items)
     if not self.ui.document then -- FileManager menu only
         menu_items.opds = {
-            text = _("OPDS catalog"),
+            text = _("OPDS catalogs"),
```

Comment:
We also have the "OPDS catalog" string in Dispatcher, and in the list title. It's an old string, maybe let it be.

Reply:
I suppose it depends on what you take it to refer to. In singular it's presumably a catalog of OPDS catalogs, though it's more of a simple list really. :-)